pennylane/templates/embeddings/amplitude.py

```diff
@@ -143,10 +143,19 @@ def __init__(self, features, wires, pad_with=None, normalize=False, pad=None, do
         features = self._preprocess(features, wires, pad_with, normalize)
         super().__init__(features, wires=wires, do_queue=do_queue)
 
+    def adjoint(self, do_queue=False):
+        return qml.templates.MottonenStatePreparation(self.parameters[0], wires=self.wires)
```

Comment:
This was an idea by @mariaschuld: if inverting AmplitudeEmbedding, we should force it to go via the Mottonen route

Reply:
I was actually thinking that even without this explicit adjoint function this would be done, since the template calls `QubitStateVector` which should not have an adjoint, and hence its decomposition is used? But great like that too!